lib/carbonite/migrations/v1.ex

```diff
@@ -10,34 +10,10 @@ defmodule Carbonite.Migrations.V1 do
 
   @type up_option :: {:carbonite_prefix, prefix()}
 
-  @impl true
-  @spec up([up_option()]) :: :ok
-  def up(opts) do
-    prefix = Keyword.get(opts, :carbonite_prefix, default_prefix())
-
-    # ---------------- Schema --------------------
-
-    execute("CREATE SCHEMA IF NOT EXISTS #{prefix}")
-
-    # -------------- Transactions ----------------
-
-    create table("transactions", primary_key: false, prefix: prefix) do
-      add(:id, :xid8, null: false, primary_key: true)
-      add(:meta, :map, null: false, default: %{})
-      add(:processed_at, :utc_datetime_usec)
-
-      timestamps(updated_at: false, type: :utc_datetime_usec)
-    end
-
-    create(
-      index("transactions", [:inserted_at],
-        where: "processed_at IS NULL",
-        prefix: prefix
-      )
-    )
-
+  @spec create_set_transaction_id(prefix()) :: :ok
+  def create_set_transaction_id(prefix) do
```

Comment:
`nitpick`

Reading this method name got me confused for a moment. Would it be worth to add a `_trigger` suffix (or similar) to the name? 

i.e `create_trigger_set_transaction_id` or `create_trigger_set_transaction_id`

Reply:
hmm rather, `_procedure`, no? The trigger itself isn't touched by this.

Reply:
Yep `_procedure` would do the job.

Reply:
done